index.js

```diff
@@ -10,10 +11,9 @@ function stringHelper () {
     const len = arguments.length
     for (let i = 0; i < len; i += 1) {
       let arg = arguments[i]
-      let type = classifier(arg)
-      if (type === 'function') this.cat(arg.apply(this))
-      else if (type === 'array') this.cat(...arg.flatten())
-      else if (type === 'undefined' || type === 'object') continue
+      if (isType('function', arg)) this.cat(arg.apply(this))
```

Comment:
using string literals among all the code is not a good practice, what about having something more semantic like
`isBoolean(value)` the linter will tell you about not defined `isBoolean` function but not that `isType('buulean', value)` will never match anything.